### DISCUSSION TITLE
How do I increase width of main coding space?

### DISCUSSION THREAD

**Discussion by lazylavenders** [fonsp/Pluto.jl#2630] [Questions about using Pluto] [answered]
Is it possible to have two rows of code side-by-side in Pluto.jl?
Or is it possible to just increase the width of coding area?

I just find a lot of space wasted with setup, and would like to make a better use of it.
[](url)
![Capture](https://github.com/fonsp/Pluto.jl/assets/68007068/1ee36151-d643-40b4-b834-d7d00e327b87)

Almost two-thirds of space is getting wasted with this arrangement, and i would be grateful if I found a way to change it.

P.S. :    Are other themes available?

**Comment by holomorphism** [comment, marked as answer]
For cell widths, see issues #400 and #748, for example.
(Theme customization feature has not yet been implemented.)

Currently, you can customize the style and increase the width by adding the following cells to the notebook.  
(If the right margin is too narrow to manipulate, increase the 80px value)

```julia
html"""
<style>
main {
	max-width: calc(100vw - 80px);
	align-self: flex-start !important;
	margin-left: 30px !important;
}
</style>
"""
```

As for the side-by-side view of the code, this may not be what you intended, but it is quite convenient to work with the same notebook open in two browser windows.
![test](https://github.com/fonsp/Pluto.jl/assets/3076585/a5f7277e-6122-416c-af9e-d0eeacf1a520)
